src/app/shared/components/historical-pill/historical-pill.component.html

```diff
@@ -0,0 +1,20 @@
+<ul class="custom-list-group">
+  <li
+    class="custom-list-group-item"
+    *ngFor="let match of matches"
+    [class.active]="typeaheadTemplateMethods.isActive(match)"
+    (click)="typeaheadTemplateMethods.selectMatch(match, $event)"
+    (mouseenter)="typeaheadTemplateMethods.selectActive(match)"
+  >
+    <span [innerHtml]="getHighlightedMatch(match, query)[0]"></span>
+    <span
+      class="custom-highlight"
+      [innerHtml]="getHighlightedMatch(match, query)[1]"
+    ></span>
+    <span [innerHtml]="getHighlightedMatch(match, query)[2]"></span>
```

Comment:
I think this would be cleaner if the dereferencing of the array [0][1][2] is inside the function call instead of in the html.

Reply:
@marklise updated now - let me know if the update looks like what you had in mind.